library/CMakeLists.txt

```diff
@@ -207,6 +223,10 @@ set_target_properties(hipblaslt PROPERTIES CXX_VISIBILITY_PRESET "hidden" VISIBI
 set_target_properties(hipblaslt PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/staging")
 set_target_propertieS(hipblaslt PROPERTIES DEBUG_POSTFIX "-d")
 
+if (WIN32 AND BUILD_CLIENTS)
```

Comment:
Cannot land things like this anymore. This is not win32 generally but related to the expectations of your build system.